### ISSUE TITLE
Scaling font sizes for bigger screens 

### ISSUE THREAD

**Issue by ammichael** [hectahertz/react-native-typography#30]
Hi there,

First thanks for this awesome lib! It has helped me a lot. One question, though:

Shouldn't it be responsible for scaling for smaller/bigger screen sizes? This is almost required when dealing with Android and even more when dealing with tablets.

Thanks!

**Comment by IMRahulSaini** [comment]
Yes, responsive font size is definitely required. Without that, we need to do some extra tricks

**Comment by hectahertz** [comment]
Hi @ammichael , thanks for the idea!

The way that I see it is that this will depend a lot on your design. You might want to scale different sections in different ways, and the breakpoints will be different for every app.

I think it's probably better to leave it to the consumer of the library, as it will probably be hard to come up with a solution that works for most cases.

I'm 100% open to suggestions though! If you find a solution that makes sense feel free to open a PR 🙂

**Comment by c-emil** [comment]
Looking at the Dynamic Type Sizes in [Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/#:~:text=for%20other%20scripts.-,Dynamic%20Type%20Sizes,-Dynamic%20Type%20provides), it offers not only the default human size but also additional sizes depending on your needs (the xLarge, xxLarge, ...). It would be nice if the library was exporting something like `humanXLarge`, `humanXXLarge`, ...

What do you think?
